web/cobrands/oxfordshire/assets.js

```diff
@@ -0,0 +1,176 @@
+(function(){
+
+if (!fixmystreet.maps) {
+    return;
+}
+
+var tilma_url = "https://tilma.staging.mysociety.org/mapserver/oxfordshire";
+var proxy_base_url = "https://tilma.staging.mysociety.org/proxy/occ/";
+if ( location.hostname === 'www.fixmystreet.com' || location.hostname == 'fixmystreet.oxfordshire.gov.uk' ) {
+    tilma_url = "https://tilma.mysociety.org/mapserver/oxfordshire";
+    proxy_base_url = "https://tilma.mysociety.org/proxy/occ/";
+}
+
+var defaults = {
+    wfs_url: tilma_url,
+    asset_type: 'spot',
+    max_resolution: 4.777314267158508,
+    geometryName: 'msGeometry',
+    srsName: "EPSG:3857",
+    body: "Oxfordshire County Council",
+    strategy_class: OpenLayers.Strategy.FixMyStreet
+};
+
+fixmystreet.assets.add(defaults, {
+    wfs_feature: "Trees",
+    asset_id_field: 'Ref',
+    attributes: {
+        feature_id: 'Ref'
+    },
+    asset_category: ["Trees"],
+    asset_item: 'tree'
+});
+
+fixmystreet.assets.add(defaults, {
+    wfs_feature: "Traffic_Lights",
+    asset_id_field: 'Site',
+    attributes: {
+        feature_id: 'Site'
+    },
+    asset_category: ["Traffic Lights (permanent only)"],
+    asset_item: 'traffic light'
+});
+
+var streetlight_select = $.extend({
+    label: "${UNITNO}",
+    fontColor: "#FFD800",
+    labelOutlineColor: "black",
+    labelOutlineWidth: 3,
+    labelYOffset: 65,
+    fontSize: '15px',
+    fontWeight: 'bold'
```

Comment:
There's something about this which makes it quite hard to determine what's going on in busy areas:

<img width="148" alt="image" src="https://user-images.githubusercontent.com/4776/84022908-0818b880-a97f-11ea-904e-de39f25fa35b.png">

I tried changing the font and outline sizes, but didn't make much difference. I think it's all the yellow. Could the asset markers perhaps be a different, more contrasting, colour? With a bigger font size too it pops a bit more:

<img width="197" alt="image" src="https://user-images.githubusercontent.com/4776/84023581-3fd43000-a980-11ea-86f8-46215800a247.png">

Reply:
I agree with the idea of adding stronger colour contrast between the number and the background (which will often be covered with dots). Or even just generally between the "selected" asset marker, and the background markers.

Blue for the dots looks ok – although we’ll need to make it’s not too easily confused with the blue "here’s your location" dot that was recently added to FMS. If we change the colour, we’ll also need update the yellow dot in the sidebar "pick an asset from the map" hint, assuming that’s still there.

Reply:
Spot the location marker:

<img width="684" alt="Screenshot 2020-06-15 at 11 56 45" src="https://user-images.githubusercontent.com/5310/84649964-525de480-aeff-11ea-8d8d-6c7a058c7d54.png">

I will try some other colours.

Reply:
The dark green from the header seems to be ok:

<img width="541" alt="Screenshot 2020-06-15 at 12 01 23" src="https://user-images.githubusercontent.com/5310/84650361-fb0c4400-aeff-11ea-94a2-0ccdfdd084dd.png">

Reply:
and against some dots:

<img width="717" alt="Screenshot 2020-06-15 at 12 02 06" src="https://user-images.githubusercontent.com/5310/84650418-18411280-af00-11ea-9796-5a0e511111ca.png">